Add index pages to support heading clickability (#3837)
opensearch-project/documentation-website: _dashboards/index.md

---
layout: default
title: OpenSearch Dashboards
nav_order: 1
has_children: false
---

# OpenSearch Dashboards

> OpenSearch Dashboards is the user interface that lets you visualize your OpenSearch data and run and scale your OpenSearch clusters. Get started with OpenSearch Dashboards today!

Reviewer: ```suggestion
OpenSearch Dashboards is the user interface that lets you visualize your OpenSearch data, and run and scale your OpenSearch clusters.
```

Reviewer: Should we point them to the playground?

## Getting started

| Concept | Description | 
|---------|-------------|

> | [OpenSearch quickstart]({{site.url}}{{site.baseurl}}/dashboards/quickstart-dashboards/) | Learn about the basic concepts and features in OpenSearch Dashboards. |

Reviewer: OpenSearch Dashboards Quickstart

| [Install and configure OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/index/) | Get started with OpenSearch Dashboards. | 

> | [Upgrade OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/upgrade-opensearch/index/) | Get the latest version of OpenSearch and OpenSearch Dashboards. |

Reviewer: I would remove this.

| [Create visualizations]({{site.url}}{{site.baseurl}}/dashboards/visualize/viz-index/) | Learn about visualizing data in OpenSearch Dashboards. |
| [Explore and query data]({{site.url}}{{site.baseurl}}/dashboards/discover/index-discover/) | Learn how to explore and query data in OpenSearch. |

## Observability

| Concept | Description | 
|---------|-------------|
| [Observability in OpenSearch Dashboards]({{site.url}}{{site.baseurl}}//observing-your-data/index/) | Observe, monitor, and secure data and improve performance across tools and workflows. |

## DevTools and APIs

| Concept | Description |
|---------|-------------|

> | [Dev Tools]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/index-dev/) | Learn about the main uses of the Dev Tools Console, running queries and API requests, and using integrated tools to inspect data. |

Reviewer: the link isn't working for me.

Author: I would remove "using integrated tools to inspect data":
```suggestion
| [Dev Tools]({{site.url}}{{site.baseurl}}/dashboards/dev-tools/index-dev/) | Learn how to run OpenSearch queries in an integrated console. |
```
